win32ss/gdi/ntgdi/freetype.c

```diff
@@ -985,6 +1048,10 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont,
         FontGDI->CharSet = SYMBOL_CHARSET;
     }
 
+    /* delete old font substitute */
+    DeleteFontSubstFromList(&FontSubstListHead, &Entry->FaceName);
```

Comment:
`FontSubstListHead` is no longer read-only after initialization, so we should synchronize access to it.
Preferrably with the `FreeTypeLock` (by using `IntLockFreeType`).

Reply:
OK.